.github/workflows/manifest.yml

```diff
@@ -26,7 +26,7 @@ jobs:
           west init -l . || true
 
       - name: Manifest
-        uses: zephyrproject-rtos/action-manifest@f223dce288b0d8f30bfd57eb2b14b18c230a7d8b
+        uses: zephyrproject-rtos/action-manifest@v1.2.0
```

Comment:
Actually no, the best practice is to use tags. See for example how we use the ubiquitous checkout action:

https://github.com/zephyrproject-rtos/zephyr/blob/3b9acef1ba5f935883260a4d09214ad11e68295a/.github/workflows/compliance.yml#L15

or

https://github.com/zephyrproject-rtos/zephyr/blob/3b9acef1ba5f935883260a4d09214ad11e68295a/.github/workflows/compliance.yml#L21

Reply:
Ya, the best practice is to "Pin actions to a tag only if you trust the creator" (https://docs.github.com/en/actions/security-guides/security-hardening-for-github-actions#using-third-party-actions). It's definitely fair to trust GitHub with their own releases, and I guess we should trust ourselves too, but I simply remember messing up a tag myself before (for first-time-contributor action), and I think the error could have been avoided more easily caught during review had I used a SHA.

More specifically, I had accidentally created a tag on the `zephyrproject-rtos/action-first-interaction` repo that was pointing to a commit that actually never was pushed to this repo, and only lived in my own fork (yes, that's possible...). Looking at the GitHub UI, it would have been very hard for a reviewer to catch the mistake, for a quick look would show the tag, and an associated commit description that would look just fine.

Reply:
> Pin actions to a tag only if you trust the creator

Well, the action is under the Zephyr org, we trust that one, right?